<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPuntuacion</class>
 <widget class="QWidget" name="WidgetPuntuacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>214</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="title_text_label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="perfect_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Perfect:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="perfect_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="perfect_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="great_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Great</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="great_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="great_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="good_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Good</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="good_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="good_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="poor_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Poor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="poor_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="poor_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="miss_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Miss</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="miss_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="miss_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="dificulty_Label">
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Dificulty</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="max_combo_Label">
     <property name="text">
      <string>Max Combo</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="max_combo_text_Label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="max_combo_per_Label">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
